### PR TITLE
Add optional parameter to syscall reboot/shutdown

### PR DESCRIPTION
**Pull request by CodeLongAndProsper90**
Added a optional parameter to tlib::shutdown and tlib::reboot that waits the number of seconds, to enable timed shutdowns/reboots

### CODE REVIEW

File: tlib/src/system.cpp [hunk 1]
```diff
@@ -99,7 +99,8 @@ std::expected<size_t> tlib::exec_and_wait(const char* executable, const std::vec
     return std::move(result);
 }
 
-void tlib::reboot(){
+void tlib::reboot(int delay=0) {
+    tlib::sleep_ms(1000*delay);
```
Comment (wichtounet) [inline tlib/src/system.cpp]: Could you add a test to do the sleep only if delay is not zero? And add spaces around *.

File: tlib/src/system.cpp [hunk 2]
```diff
@@ -108,7 +109,8 @@ void tlib::reboot(){
     __builtin_unreachable();
 }
 
-void tlib::shutdown(){
+void tlib::shutdown(int delay=0){
+    tlib::sleep_ms(1000* delay);
```
Comment (wichtounet) [inline tlib/src/system.cpp]: Could you add a test to do the sleep only if delay is not zero? And add a space before *.

**Review by wichtounet — CHANGES_REQUESTED** [review]
That's a good idea :+1: 

Just two small things to address in the code and I will merge your contribution :)

**Review by wichtounet — APPROVED** [review]

**Comment by wichtounet** [comment]
Thanks for your contribution :+1:
